samza-log4j/src/main/java/org/apache/samza/logging/log4j/StreamAppender.java

```diff
@@ -275,6 +278,8 @@ protected Config getConfig() {
     } catch (IOException e) {
       throw new SamzaException("can not read the config", e);
     }
+    // Make system producer drop producer errors for StreamAppender
+    config = new MapConfig(config, ImmutableMap.of(TaskConfig.DROP_PRODUCER_ERROR(), "true"));
```

Comment:
@PawasChhokra 
I think it will be better to add a assertion in existing  `StreamAppender` tests to check if this config property is turned on.

Reply:
Yes. Got it.